styles/base.css

```diff
@@ -0,0 +1,185 @@
+@import url(https://fonts.googleapis.com/css?family=Open+Sans:400,300,800italic,800,700italic,700,600italic,600,400italic,300italic);
```

Comment:
Delete this page. CSS has been migrated and this stylesheet is no longer linked to index.html page.